test/unit/gen/challenge.go

```diff
@@ -128,6 +129,12 @@ func SetChallengeDeletionTimestamp(ts metav1.Time) ChallengeModifier {
 	}
 }
 
+func SetChallengeUID(uid string) ChallengeModifier {
```

Comment:
What is this function used for?

Reply:
Ah that was a remnant from my old controller approach.. will clean that up its not required anymore